src/tracer/MOM_generic_tracer.F90

```diff
@@ -348,6 +398,8 @@ subroutine initialize_MOM_generic_tracer(restart, day, G, GV, US, h, param_file,
         call set_initialized(tr_ptr, g_tracer_name, CS%restart_CSp)
       endif
 
+      call g_tracer_get_obc_segment_props(g_tracer,g_tracer_name,obc_has )
+      if(obc_has .and. g_tracer_is_prog(g_tracer)) call fill_obgc_segments(G, GV, CS%OBC, tr_ptr, g_tracer_name)
```

Comment:
Replace `CS%OBC` with OBC. `CS%OBC` is the unneeded generic tracer OBC and is null.